snap/snapcraft.yaml

```diff
@@ -5,12 +5,76 @@ license: BSD-3-Clause
 grade: stable
 confinement: strict
 contact: solutions-engineering@lists.canonical.com
-summary: DERP Relay Server for Tailscale
+issues: https://github.com/canonical/derper-snap/issues
+summary: DERP Server for Tailscale
 description: |
-  Self hosted DERP Relay servers for Tailscale networks.
-  This snap provides a simple way to run your own DERP servers to route your tailscale traffic through your own relay servers.
+  Derper is a self hosted DERP (Designated Encrypted Relay for Packets) server, designed to be used with Tailscale.
```

Comment:
non-blocking: I would prefer following the Diataxis in some form for cohesion between the projects

Reply:
In my view, the current description might be too lengthy.

The snapcraft.yaml description appears when users run snap info <snap-name>. After reviewing some popular snaps like MicroK8s, Terraform, and LXD, I suggest condensing the content here and focusing on the essentials.

Additional details could be moved to a separate document for users who need more in-depth information.

Reply:
Agreed, it's a lot in one place. Do you have suggestions for where to move the additional details? The snap store doesn't have an equivalent to discourse based docs like charmhub.

Reply:
Until snap store has some thing equivalent to discourse page like charmhub, I think it's fine to have summary in the description, and add a link to the github readme (or something like) for detail description following Diataxis in some form

Reply:
> Agreed, it's a lot in one place. Do you have suggestions for where to move the additional details? The snap store doesn't have an equivalent to discourse based docs like charmhub.

I more prefer separate document on github. Only put a link on description.